lightning/src/routing/gossip.rs

```diff
@@ -269,9 +285,11 @@ impl<L: Deref> EventHandler for NetworkGraph<L> where L::Target: Logger {
 						self.channel_failed(short_channel_id, is_permanent);
 					},
 					NetworkUpdate::NodeFailure { ref node_id, is_permanent } => {
-						let action = if is_permanent { "Removing" } else { "Disabling" };
-						log_debug!(self.logger, "{} node graph entry for {} due to a payment failure.", action, node_id);
-						self.node_failed(node_id, is_permanent);
+						if is_permanent {
+							log_debug!(self.logger,
+								"Removed node graph entry for {} due to a payment failure.", node_id);
```

Comment:
Should we use `log_pubkey` here? Was never sure when that's necessary or if we've been consistent.